docs/getting-started/installation-with-esm.md

```diff
@@ -157,3 +151,107 @@ code of our product packages in `highcharts/es-modules/masters` for details.
 *Note:* Highcharts extensions and more advanced series might not be ready yet
 for ES6 module loading. In this case you have to use one of the product
 packages.
+
+
+## Dynamic imports from CDN
+
+Highcharts is available on our CDN as ECMAScript modules. You can [import ES modules directly in modern browsers](https://jakearchibald.com/2017/es-modules-in-browsers/)
+without any bundling tools by using `<script type="module">` ([demo](https://jsfiddle.net/highcharts/rtcx6j3h/)):
```

Comment:
We have samples demos form loading ESM in the repo, see `highcharts/esm/*` . Use one of those in the docs, because raw jsfiddle links are hard to maintain. Ref #17252.